tests/allotrope/schema_parser/generate_schemas_test.py

```diff
@@ -5,4 +5,7 @@
 
 def test_generate_schemas_runs_to_completion() -> None:
     root_dir = Path(__file__).parent.parent.parent.parent
-    assert generate_schemas(root_dir) == 10
+    models_changed = generate_schemas(root_dir, dry_run=True)
+    assert (
+        not models_changed
+    ), f"Expected no models files to have changed by generate-schemas script, found changes in: {models_changed}. Please run scripts:generate-schemas and validate the changes."
```

Comment:
maybe we can have the full line to run (`hatch run scripts:generate-schemas`) in the error message so people can just copy-paste?

Reply:
Done!